app/Events/RoleGroupPivotDeleted.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Events;
+
+use App\Models\Pivots\RoleGroupPivot;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Broadcasting\PrivateChannel;
+use Illuminate\Foundation\Events\Dispatchable;
+use Illuminate\Broadcasting\InteractsWithSockets;
+
+class RoleGroupPivotDeleted
+{
+    use Dispatchable;
+    use InteractsWithSockets;
+    use SerializesModels;
+
+    /**
+     * Create a new event instance.
+     */
+    public function __construct(
+        public RoleGroupPivot $roleGroupPivot
+    ) {
+    }
+
+    /**
+     * Get the channels the event should broadcast on.
+     */
+    public function broadcastOn(): array
+    {
+        return [
+            new PrivateChannel('channel-name'),
+        ];
+    }
```

Comment:
Feel free to ignore cleanup tasks if your intention is to do so later. But I believe this can be removed if we are not working with broadcasting.

Reply:
Will be removed upon cleanup.